test/databind-with-unknown-property/databind-with-unknown-property.html

```diff
@@ -6,13 +6,34 @@
     <div id="{{method(writtenTo)}}"></div>
     <div title="{{title}}"></div>
 
+    <dom-repeat>
+      <template>
+        <div id="{{item}}" title="{{index}}"></div>
+      </template>
+    </dom-repeat>
+
+    <dom-repeat as="smeerp" indexAs="gavagai">
+      <template>
+        <div id="{{smeerp}}" title="{{gavagai}}"></div>
+      </template>
+    </dom-repeat>
+
+    <template is='dom-repeat'>
+      <div id="{{item}}" title="{{index}}"></div>
+    </template>
+
+    <template is='dom-repeat' as="tribble" indexAs="rabbit">
+      <div id="{{tribble}}" title="{{rabbit}}"></div>
+    </template>
+
     <!-- Bad -->
     <div id="{{horse}}"></div>
     <div id="[[onlyReadFrom]]"></div>
     <p id="[[onlyReadFrom]]"></p>
     <nav id="[[onlyReadFrom]]"></nav>
 
```

Comment:
Maybe a `<div id="{{rabbit}}">` out here would be worthwhile check?  You want to ensure that a scoped index isn't misperceived as a template-only property definition just because it appeared as a two-way binding here?